inspector_pause test flaky

https://github.com/denoland/deno/runs/555522900
```
---- inspector_pause stdout ----
target_dir D:\a\deno\deno\target\release
ws_url ws://127.0.0.1:9230/ws/32db3e15-0d08-4ee4-af90-b3f9ca5a7273
##[error]thread 'inspector_pause' panicked at 'called `Result::unwrap()` on an `Err` value: Io(Os { code: 10054, kind: ConnectionReset, message: "An existing connection was forcibly closed by the remote host." })', cli\tests\integration_tests.rs:2076:17
```

--- a/cli/tests/integration_tests.rs
+++ b/cli/tests/integration_tests.rs
@@ -2015,7 +2015,6 @@ fn extract_ws_url_from_stderr(
   url::Url::parse(ws_url).unwrap()
 }
 
-#[cfg(not(target_os = "linux"))] // TODO(ry) broken on github actions.
 #[tokio::test]
 async fn inspector_connect() {
   let script = deno::test_util::root_path()
@@ -2122,7 +2121,6 @@ async fn inspector_break_on_first_line() {
   child.kill().unwrap();
 }
 
-#[cfg(not(target_os = "linux"))] // TODO(ry) broken on github actions.
 #[tokio::test]
 async fn inspector_pause() {
   let script = deno::test_util::root_path()
@@ -2182,7 +2180,6 @@ async fn inspector_pause() {
   child.kill().unwrap();
 }
 
-#[cfg(not(target_os = "linux"))] // TODO(ry) broken on github actions.
 #[tokio::test]
 async fn inspector_port_collision() {
   let script = deno::test_util::root_path()

